al5d_gazebo/scripts/arm_controller.py

```diff
@@ -61,8 +67,17 @@ def state_cb(self, state):
             self.status_q.get()
         self.status_q.put((self.pos, self.vel))
 
+    def model_cb(self, msg):
+        self.model_data = msg;
+
+    def opponent_cb(self, msg):
+        self.q_opponent = msg.position;
+        self.qd_opponent = msg.velocity;
+
     def contact_cb(self, msg):
 
+        # THIS IS PROBABLY BROKEN BY NAMESPACING
```

Comment:
Are we concerned that this is broken and if so do we want to fix it at a later point in time?

Reply:
Tbh, I think we should eliminate is_collided from the ArmController, but since it's out there being used now I guess we should keep it in til the end of this semester. I will try and fix this for the namespaced case. This issue shouldn't affect prior labs but students might possibly want to use this in the final project. tho I wouldn't recommend it

Reply:
Should be resolved in next commit